### PR TITLE
RIP Slack integration

### PR DESCRIPTION
**Pull request by asdfryan**
Remove all slack integration code in preparation for discord.

### CODE REVIEW

**Review by rawxfish — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -57,17 +57,6 @@ As a hunt progresses, the number of puzzles can get quite large (especially for
...
-
-![answer-queue](https://user-images.githubusercontent.com/544734/71759910-179d4e00-2e83-11ea-94bc-f26dd46a8a31.png)
-
-Generally, a dedicated person will monitor this Answer Queue page and submit the guesses on the actual hunt website, updating the status to SUBMITTED once submitted on the hunt site. When the team gets notified whether the answer is right or wrong, the status can be updated to CORRECT or INCORRECT. Once an answer has been marked CORRECT, the puzzle will be marked SOLVED on the main puzzles page and its status can no longer be updated there. In case this was a mistake, the status of the answer can be changed to INCORRECT to allow updates to the puzzle status on the main puzzles page.
-
-Notes (such as feedback that our guess was a partial answer) may be entered in the "Notes" field, and these will automatically be posted in the puzzle's Slack channel as well.
```
Comment (erwa) [inline README.md:68]: I think some of this info is still relevant -- you can just remove the Slack parts and keep the rest

**Review by erwa — APPROVED** [review]
Looks good to me. Wow, there was Slack-related logic everywhere!

File: README.md (hunk 1 above)
Comment (rawxfish) [reply README.md:68]: The plan is to remove answer queue so the answer input and status flow will change as well, so I think this info will end up being irrelevant anyways

Comment (asdfryan) [reply README.md:68]: +1 that was my idea

Comment (erwa) [reply README.md:68]: Oh, gotcha, makes sense to remove then.

**Pull request merged**